spec/classes/foreman_service_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe 'foreman::service' do
   let :facts do
-    on_supported_os['redhat-8-x86_64']
+    on_supported_os['redhat-9-x86_64']
```

Comment:
I wonder if we could reuse `DEFAULT_OS_FACTS` from `spec/support/plugin.rb`.

Reply:
Would this work?
```suggestion
    on_supported_os.first.last
```

Reply:
yes that works